apps/worker/src/index.ts

```
@@ -1,6 +1,7 @@
 import { Hono } from 'hono';
 import { cors } from 'hono/cors';
 import { Env } from './types/env';
+import { HonoEnv } from './types/hono';
```

ESLint

[error] 'HonoEnv' is defined but never used.

(no-unused-vars)